### DISCUSSION TITLE
Welcome to indonesian-language-models Discussions!

### DISCUSSION THREAD

**Discussion by cahya-wirawan** [cahya-wirawan/language-modeling#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JemmyRohmadWaluyo** [comment]
Selamat Sore, saya mau bikin chatbot dalam Bahasa Indonesia, saya benar-benar newbie. Kira-kira ada arahan ga step2 nya apa yang harus dipelajari, terima kasih

**Comment by hrizal** [comment]
mas Cahya, apakah model nya bisa digunakan di gpt4all ?

**Reply by cahya-wirawan** [reply]
Ini saya mau coba. Nanti saya kabari hasilnya

**Reply by hrizal** [reply]
Alhamdulillah kereeen
